Makefile

```diff
@@ -61,6 +69,7 @@ fmt: fmt-nix fmt-rust fmt-js
 .PHONY: fmt-check
 fmt-check: fmt-nix-check fmt-rust-check fmt-js-check
 
+# Clippy builds the CLI, in order to build the CLI we need to build the kernel
 .PHONY: lint
-lint:
+lint: build-kernel
```

Comment:
Clippy must expand all macros _before_ linting. Since `jstz_kernel.wasm` is a build-time dependency (explicitly required for expanding macros), then we must build the kernel before linting the CLI